addOns/pscanrulesBeta/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Changed
 - Replace usage of CWE-200 for the In Page Banner Information Leak scan rule (Issue 8731).
+- Add support for 'credentialless' COEP value in Site Isolation Scan (Issue 8840).
```

Comment:
The scan rule is called "Insufficient Site Isolation Against Spectre Vulnerability", also better to use directive instead of value.